packages/ui/src/components/molecules/Info.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+
+import { InfoIcon } from '../../assets';
+import { Container, LangDisplay, Typography } from '../atoms';
+
+export const Info: FC<{
```

Comment:
From the name, it's not clear what the component does and where it should be used.